sunpy/map/sources/gong.py

```diff
@@ -54,8 +51,20 @@ def is_datasource_for(cls, data, header, **kwargs):
 
     @property
     def date(self):
+        # The FITS file has a date that is made from the date-obs and time-obs keywords
+        # Which is not what date-obs is supposed to be
         return Time(f"{self.meta.get('date-obs')} {self.meta.get('time-obs')}")
 
+    def _set_date(self, date):
+        self.meta['date-obs'], self.meta['time-obs'] = parse_time(date).utc.iso.split(' ')
```

Comment:
This is insanely picky, but using T feels like a better separator to me.
```suggestion
        self.meta['date-obs'], self.meta['time-obs'] = parse_time(date).utc.isot.split('T')
```

Reply:
Heh, picky indeed!  If a wayward space were to creep into the `TimeISO` format, I think the chaos would extend far beyond this one line.